pyfstat/core.py

```diff
@@ -1180,23 +1222,63 @@ def get_transient_maxTwoFstat(
             self.windowRange.tau = int(tend - tstart)  # TYPE UINT4
 
         self.FstatMap, self.timingFstatMap = tcw.call_compute_transient_fstat_map(
-            self.tCWFstatMapVersion,
-            self.tCWFstatMapFeatures,
-            self.FstatResults.multiFatoms[0],  # 0 index: single frequency bin
-            self.windowRange,
+            version=self.tCWFstatMapVersion,
+            features=self.tCWFstatMapFeatures,
+            multiFstatAtoms=self.FstatResults.multiFatoms[0],  # single frequency bin
+            windowRange=self.windowRange,
+            BtSG=getattr(self, "BtSG", False),
         )
 
-        # Now instead of the overall twoF,
-        # we get the maximum twoF over the transient window range.
+        # get the maximum twoF over the transient window range
         self.maxTwoF = 2 * self.FstatMap.maxF
         if np.isnan(self.maxTwoF):
             self.maxTwoF = 0
+
+        if getattr(self, "BtSG", False):
```

Comment:
```suggestion
        if self.BtSG:
```

Reply:
See above, needed to make semicoherent classes to run without explicitly setting it to False in those too.